pkgs/tools/misc/ttfautohint/qt510.patch

```diff
@@ -0,0 +1,11 @@
+--- a/configure	2017-12-29 01:10:59.201970193 +0100
++++ b/configure	2017-12-29 01:10:40.161330131 +0100
```

Comment:
It's unfortunate that this patches `configure` rather than `configure.ac` from which it was generated.

Reply:
Judging by the fact that ttfautohint `configure.ac` doesn't have `qmake_version_sed` at all, it's brought in by Autotools and this could be solved just by autoreconfing the package. Could you please try adding `autoreconfHook` to `nativeBuildInputs`?

(We should really autoreconf all Autotools packages).

Reply:
I tried that, but that does not work.